README.md

```diff
@@ -291,14 +291,32 @@ Optional parameters:
 - `password`: String, password to an encrypted document
 - `config`: Config, viewer configuration options
 
+configs (more info could be found in `lib/config.dart`):
+
+Name | Type | Default | Explanation
+-- | -- | -- | -- | 
+disabledElements | array of `Buttons` constants | empty | Buttons to be disabled for the viewer
+disabledTools | array of `Tools` constants | empty | Tools to be disabled for the viewer
+multiTabEnabled | boolean | false | enable document multi-tab mode
+customerHeaders | map<string, string> | empty | custom headers to use with HTTP/HTTPS requests
+fitMode | `FitModes` constant | `FitModes`.`fitWidth` | the fit mode for the document window
+layoutMode | `LayoutModes` constant | `LayoutModes`.`continuous` | the layout mode for the document window
```

Comment:
Can you link to where they can find the valid constants these are referring to? eg. ```[`Button`](#section-with-buttons-listed]```